internal/common/disasters/environmentdefinition.go

```diff
@@ -69,3 +69,21 @@ func (e Environment) DisasterEffects() map[shared.ClientID]float64 {
 	}
 	return out
 }
+
+// DisasterEffects returns the porportional effects of the most recent DisasterReport held in the environment state
+func (e Environment) DisasterPorpotionalEffects() map[shared.ClientID]float64 {
+	out := map[shared.ClientID]float64{}                         // TODO: change key type to ClientID
+	porportionalEffect := map[shared.ClientID]float64{}
+	totalEffect := 0.0   
+
+	epiX, epiY := e.lastDisasterReport.x, e.lastDisasterReport.x // epicentre of the disaster (peak mag)
```

Comment:
```suggestion
	epiX, epiY := e.lastDisasterReport.x, e.lastDisasterReport.y // epicentre of the disaster (peak mag)
```
this would've been painful to debug

Reply:
nice catch